### PR TITLE
v6.5: add lock stats docs

### PR DESCRIPTION
**Pull request by xuyifangreeneyes**
### What is changed, added or deleted? (Required)

1. Add system variable `tidb_auto_build_stats_concurrency` and `tidb_sysproc_scan_concurrency`.
2. Add the description of lock/unlock statistics.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by chrysan — APPROVED** [review]

File: sql-statements/sql-statement-lock-stats.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: LOCK STATS
+summary: TiDB 数据库中 LOCK STATS 的使用概况。
+aliases: ['/docs-cn/dev/sql-statements/sql-statement-lock-stats/']
```
Comment (TomShawn) [inline sql-statements/sql-statement-lock-stats.md:4]:
```suggestion
```

File: sql-statements/sql-statement-lock-stats.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+1 row in set (0.00 sec)
+```
+
+## MySQL 兼容性
+
+该语句是 TiDB 对 MySQL 语法的扩展。
+
+## 另请参阅
+
+* [统计信息](/statistics.md)
```
Comment (TomShawn) [inline sql-statements/sql-statement-lock-stats.md:103]:
```suggestion
* [统计信息](/statistics.md#锁定统计信息)
```

File: sql-statements/sql-statement-show-stats-locked.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
+---
+title: SHOW STATS_LOCKED
+summary: TiDB 数据库中 SHOW STATS_LOCKED 的使用概况。
+aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-stats-locked/']
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-stats-locked.md:4]:
```suggestion
```

File: sql-statements/sql-statement-show-stats-locked.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+---
+
+# SHOW STATS_LOCKED
+
+`SHOW STATS_LOCKED` 语句显示统计信息锁定的表。
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-stats-locked.md:9]:
```suggestion
`SHOW STATS_LOCKED` 语句显示统计信息被锁定的表。
```

File: sql-statements/sql-statement-lock-stats.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
+---
+title: LOCK STATS
+summary: TiDB 数据库中 LOCK STATS 的使用概况。
+aliases: ['/docs-cn/dev/sql-statements/sql-statement-lock-stats/']
+---
+
+# LOCK STATS
+
+`LOCK STATS` 语句用于锁定统计信息。
```
Comment (TomShawn) [inline sql-statements/sql-statement-lock-stats.md:9]:
```suggestion
`LOCK STATS` 语句用于锁定一张表上的统计信息。
```

File: sql-statements/sql-statement-unlock-stats.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
+---
+title: UNLOCK STATS
+summary: TiDB 数据库中 UNLOCK STATS 的使用概况。
+aliases: ['/docs-cn/dev/sql-statements/sql-statement-unlock-stats/']
+---
+
+# UNLOCK STATS
+
+`UNLOCK STATS` 语句用于解锁统计信息。
```
Comment (TomShawn) [inline sql-statements/sql-statement-unlock-stats.md:9]:
```suggestion
`UNLOCK STATS` 语句用于解锁统计信息被锁定的表。
```

File: sql-statements/sql-statement-unlock-stats.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+1 row in set (0.00 sec)
+```
+
+## MySQL 兼容性
+
+该语句是 TiDB 对 MySQL 语法的扩展。
+
+## 另请参阅
+
+* [统计信息](/statistics.md)
```
Comment (TomShawn) [inline sql-statements/sql-statement-unlock-stats.md:103]:
```suggestion
* [统计信息](/statistics.md#锁定统计信息)
```

File: statistics.md [hunk 8]
```diff
@@ -728,7 +728,84 @@ LOAD STATS 'file_name';
...
++-------+------+-----------------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
 ## 另请参阅
 
 * [LOAD STATS](/sql-statements/sql-statement-load-stats.md)
 * [DROP STATS](/sql-statements/sql-statement-drop-stats.md)
+* [LOCK STATS](/sql-statements/sql-statement-lock-stats.md)
+* [UNLOCK STATS](/sql-statements/sql-statement-unlock-stats.md)
```
Comment (TomShawn) [inline statistics.md:811]:
```suggestion
* [UNLOCK STATS](/sql-statements/sql-statement-unlock-stats.md)
* [SHOW STATS_LOCKED](/sql-statements/sql-statement-show-stats-locked.md)
```

File: system-variables.md [hunk 9]
```diff
@@ -619,6 +619,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
     - `tidb_auto_analyze_start_time='01:00 +0000'`
     - `tidb_auto_analyze_end_time='03:00 +0000'`
 
+### `tidb_auto_build_stats_concurrency` <span class="version-mark">从 v6.5.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 默认值：`1`
+- 范围：`[1, 256]`
+- 这个变量用来设置统计信息自动更新执行时并发度。
```
Comment (TomShawn) [inline system-variables.md:628]:
```suggestion
- 这个变量用来设置执行统计信息自动更新的并发度。
```

File: system-variables.md [hunk 10]
```diff
@@ -3106,6 +3114,14 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 默认值：`OFF`
 - 这个变量用于控制 `SYSDATE` 函数能否替换为 `NOW` 函数，其效果与 MYSQL 中的 [`sysdate-is-now`](https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_sysdate-is-now) 一致。
 
+### `tidb_sysproc_scan_concurrency` <span class="version-mark">从 v6.5.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 默认值：`1`
+- 范围：`[1, 256]`
+- 这个变量用来设置内部 SQL（比如统计信息自动更新）的 scan 操作的并发度。
```
[the remark this answers is not in the document]
Comment (xuyifangreeneyes) [reply system-variables.md:3123]: 内部 SQL 就是 tidb 内部会执行一些 SQL 用来维护统计信息、ddl 之类的信息，与之对应的是用户执行的 SQL。

[the remark this answers is not in the document]
Comment (xuyifangreeneyes) [reply system-variables.md:3123]: 统计信息自动更新就是通过 tidb 内部执行 analyze table t 语句来完成的。

File: sql-statements/sql-statement-show-stats-locked.md [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+mysql> show warnings;
++-------+------+-----------------------------------------------------------------+
+| Level | Code | Message                                                         |
++-------+------+-----------------------------------------------------------------+
+| Note  | 1105 | Analyze use auto adjusted sample rate 1.000000 for table test.t |
++-------+------+-----------------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+
+锁定表 `t` 的统计信息，执行 `ANALYZE` 语句，使用 `SHOW STATS_LOCKED` 可查看表 `t` 的统计信息已被锁定。 warning 提示跳过对表 `t` 的 `ANALYZE`。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-show-stats-locked.md:46]:
```suggestion
锁定表 `t` 的统计信息，执行 `ANALYZE` 语句，使用 `SHOW STATS_LOCKED` 可查看表 `t` 的统计信息已被锁定。warning 提示跳过对表 `t` 的 `ANALYZE`。
```

File: system-variables.md [hunk 12]
```diff
@@ -3170,6 +3178,14 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 默认值：`OFF`
 - 这个变量用于控制 `SYSDATE` 函数能否替换为 `NOW` 函数，其效果与 MYSQL 中的 [`sysdate-is-now`](https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_sysdate-is-now) 一致。
 
+### `tidb_sysproc_scan_concurrency` <span class="version-mark">从 v6.5.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
+- 默认值：`1`
+- 范围：`[1, 256]`
+- 这个变量用来设置 TiDB 执行内部 SQL 语句（例如统计信息自动更新） 时 scan 操作的并发度。
```
Comment (Oreoxmt) [inline system-variables.md:3187]:
```suggestion
- 这个变量用来设置 TiDB 执行内部 SQL 语句（例如统计信息自动更新）时 scan 操作的并发度。
```

File: system-variables.md [hunk 13]
```diff
@@ -3170,6 +3178,14 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 默认值：`OFF`
 - 这个变量用于控制 `SYSDATE` 函数能否替换为 `NOW` 函数，其效果与 MYSQL 中的 [`sysdate-is-now`](https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_sysdate-is-now) 一致。
 
+### `tidb_sysproc_scan_concurrency` <span class="version-mark">从 v6.5.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
```
Comment (Oreoxmt) [inline system-variables.md:3184]:
```suggestion
- 是否持久化到集群：是
- 类型：整数
```

File: system-variables.md [hunk 14]
```diff
@@ -654,6 +654,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
     - `tidb_auto_analyze_start_time='01:00 +0000'`
     - `tidb_auto_analyze_end_time='03:00 +0000'`
 
+### `tidb_auto_build_stats_concurrency` <span class="version-mark">从 v6.5.0 版本开始引入</span>
+
+- 作用域：GLOBAL
+- 是否持久化到集群：是
```
Comment (Oreoxmt) [inline system-variables.md:660]:
```suggestion
- 是否持久化到集群：是
- 类型：整数
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by TomShawn — APPROVED** [review]

**Pull request merged**